erpnext_documentation/www/docs/user/manual/en/manufacturing/routing.md

```diff
@@ -29,6 +29,12 @@ To access the Routing list, go to:
 Once created, a Routing can be selected in a BOM to fetch the Operations stored in the Routing.
 ![Routing BOM](/docs/assets/img/manufacturing/routing-bom.png)
 
+## 3. Sequence ID in Routing
+![Routing Sequence Id](/docs/assets/img/manufacturing/sequence-id-routing.png)
+Sequence Id is enforce the users to complete the operations sequentially. In case if the user has tried to complete the last operation first before completions of other operations using Job Card then based on Sequence ID system throws the validation error.
```

Comment:
```suggestion
Sequence ID enforces the users to complete the operations sequentially via Job Card. In case a user tries to complete an operation before completing any of its precedent operations as per the Sequence ID, the system throws a validation error as shown in the screenshot below.
```